toolchain/codegen/codegen.h

```diff
@@ -0,0 +1,22 @@
+// Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+// Exceptions. See /LICENSE for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#ifndef CARBON_TOOLCHAIN_CODEGEN_CODEGEN_H_
+#define CARBON_TOOLCHAIN_CODEGEN_CODEGEN_H_
+
+#include <cstdint>
+
+#include "llvm/IR/Module.h"
+
+namespace Carbon {
+inline bool codegen_has_errors = false;
```

Comment:
This shouldn't be here. Globals are generally undesirable (they prevent multi-threaded execution), but also it's only used locally inside PrintAssemblyFromModule.